src/core/config/transformer/preset.rs

```diff
@@ -5,14 +5,25 @@ use crate::core::transform::{self, Transform, TransformerOps};
 
 /// Defines a set of default transformers that can be applied to any
 /// configuration to make it more maintainable.
-#[derive(Setters)]
+#[derive(Setters, Debug, PartialEq)]
 pub struct Preset {
     merge_type: f32,
     consolidate_url: f32,
     tree_shake: bool,
     use_better_names: bool,
 }
 
+impl Preset {
+    pub fn new(
+        merge_type: f32,
+        consolidate_url: f32,
+        tree_shake: bool,
+        use_better_names: bool,
+    ) -> Self {
+        Self { merge_type, consolidate_url, tree_shake, use_better_names }
+    }
+}
```

Comment:
Not sure if I will ever use this API if the `default` operator exists. It more comfortable to use `default().merge_type(1).consolidate_url(0.5)` etc. Than using `Preset::new(1, 0.5)` and then have to forcefully pass the others parameters. Also it's easy to make a mistake of swapping the intended values of merge_type with consolidate_url since both are f32.